conductor/tests/test_async_reply_on_idle_timeout.py

```
@@ -0,0 +1,222 @@
+            "conductor-ops", "hi", profile="work", wait_for_reply=True,
+        )
+    assert ok is False
+    assert response == ""
+    assert still_running is True
+
+
+def test_wait_genuine_failure_is_not_still_running():
+    with mock.patch(
+        "bridge.run_cli", return_value=_completed(1, stderr="session not found"),
+    ):
+        ok, response, still_running = send_to_conductor(
```

Ruff (0.15.15)

[warning] Unpacked variable `response` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)